apps/ssr-tests-v9/src/utils/renderToHTML.ts

```diff
@@ -25,7 +25,7 @@ export async function renderToHTML(config: RenderToHTMLConfig): Promise<void> {
   }
 
   const resultHTML = ReactDOM.renderToString(
-    React.createElement(RendererProvider, { renderer }, React.createElement(App)),
+    React.createElement(RendererProvider, { renderer, children: React.createElement(App) }),
```

Comment:
`RendererProvider` has required `children` prop thus children need to be set within 2nd (props argument). While in react space both are valid - typescript is explicit about this